lib/commands/doctor/checks/install.js

```diff
@@ -13,28 +13,39 @@ const isRoot = require('path-is-root');
 const errors = require('../../../errors');
 const cliPackage = require('../../../../package');
 
-function checkDirectoryAndAbove(dir) {
+function checkDirectoryAndAbove(dir, extra) {
     if (isRoot(dir)) {
         return;
     }
 
-    let stats = fs.lstatSync(dir);
-    let mode = new Mode(stats);
+    return fs.lstat(dir).then((stats) => {
+        let mode = new Mode(stats);
 
-    if (!mode.others.read) {
-        throw new errors.SystemError(
-            `The path ${dir} is not readable by other users on the system.${eol}` +
-            'This can cause issues with the CLI, please either make this directory ' +
-            'readable by others or install in another location.'
-        );
-    }
+        if (!mode.others.read) {
+            return Promise.reject(new errors.SystemError(
+                `The path ${dir} is not readable by other users on the system.${eol}` +
+                'This can cause issues with the CLI, please either make this directory ' +
+                `readable by others or ${extra} in another location.`
+            ));
+        }
 
-    return checkDirectoryAndAbove(path.join(dir, '../'));
+        return checkDirectoryAndAbove(path.join(dir, '../'), extra);
+    });
 }
 
 module.exports = [{
     title: 'Checking system Node.js version',
-    task: () => {
+    task: (ctx) => {
```

Comment:
That's definitely possible - however if a user gets into this state and doesn't trigger the sqlite message, updates to ghost-cli wouldn't take effect.

Example:
- user installs ghost-cli globally
- user installs nvm after that
- user sets up install w/systemd (which makes the "ghost" system user use the globally-installed ghost-cli version
- user installs new ghost-cli version (when will end up in the nvm directory)
- systemd service file still gets old ghost-cli version, with any potential bugs/missing features, etc.